src/EditCocktail/EditCocktail.js

```diff
@@ -0,0 +1,169 @@
+import {
+  useDisclosure,
+  Button,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalCloseButton,
+  ModalHeader,
+  ModalBody,
+  FormControl,
+  FormLabel,
+  Input,
+  ModalFooter,
+} from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+import "./EditCocktail.css";
+import Ingredients from "../Ingredients/Ingredients";
+import Steps from "../Steps/Steps";
+
+const EditCocktail = ({ choosenCocktail, updateCocktail, updateSteps }) => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  const initialRef = React.useRef(null);
+  const finalRef = React.useRef(null);
+
+  const [coctailName, setCocktailName] = useState("");
+  const [newIngredient, setNewIngredient] = useState("");
+  const [newQuantity, setNewQuantity] = useState("");
+  const [newUnit, setNewUnit] = useState("");
+  const [newStep, setNewStep] = useState("");
+  const [allIngredients, setIngredients] = useState([]);
+  const [steps, setSteps] = useState([]);
+
+  useEffect(() => {
+    setIngredients(choosenCocktail.ingredients);
+    const stepsArray = choosenCocktail.steps
+      .split(" and")
+      .join(",")
+      .split(", ");
+    setSteps(stepsArray);
+  }, []);
+
+  const handleChange = (event) => {
```

Comment:
@letakeane can you review this handleChange function because I'm not sure if it is the best way to do it!

Reply:
Look into using a switch statement! It'll clean up the syntax a little bit!